### PR TITLE
fix benchmarking results

### PR DESCRIPTION
**Pull request by gatsbyz**

### CODE REVIEW

File: .github/workflows/deploy.nightly.devnet.yml [hunk 1]
```diff
@@ -125,24 +141,39 @@ jobs:
                           --premine 0xA39Fed214820cF843E2Bcd6cA1759257a530894B:1000000000000000000000000000 \
                           --premine 0x181d9fEc79EC674DD3cB30dd9dd4188E737939FE:1000000000000000000000000000 \
                           --premine 0x1AB8C3df809b85012a009c0264eb92dB04eD6EFa:1000000000000000000000000000 \
-              --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
+                          --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
+                          --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
```
Comment (Stefan-Ethernal) [inline .github/workflows/deploy.nightly.devnet.yml]: Is one of those redundant?
```suggestion
                          --block-gas-limit {{ block_gas_limit }} --block-time {{ block_time }}s \
```

File: .github/workflows/deploy.nightly.devnet.yml [hunk 2]
```diff
@@ -125,24 +141,39 @@ jobs:
...
+                          --native-token-config MyToken:MTK:18:true
 
-              polycli wallet create --words 12 --language english | jq '.Addresses[0]' > rootchain-wallet.json
+          {% if (enable_eip_1559) %}
+              # EIP-1559
+              jq --argjson code $(cat /var/lib/bootstrap/core-contracts/artifacts/contracts/child/EIP1559Burn.sol/EIP1559Burn.json)  \
+                --arg balance "$BALANCE" \
+                --arg addr "$BURN_CONTRACT_ADDRESS" \
+                '.genesis.alloc += {($addr): {"code": $code.deployedBytecode, "balance": $balance}}' \
+                genesis.json | jq . > tmp.json && mv tmp.json genesis.json
```
Comment (Stefan-Ethernal) [inline .github/workflows/deploy.nightly.devnet.yml]: Is the `EIP1559Burn` contract required in nightly builds? I wasn't aware that it is. If not, maybe consider removing it and simplifying workflow.

File: .github/workflows/deploy.nightly.devnet.yml [hunk 3]
```diff
@@ -116,6 +118,20 @@ jobs:
...
+              curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+              apt-get install -y nodejs
+
+              pushd /opt/polygon-edge/
+              make compile-core-contracts
+              cp -r /opt/polygon-edge/core-contracts /var/lib/bootstrap/core-contracts/
+              popd
+
+              BURN_CONTRACT_ADDRESS=0x0000000000000000000000000000000000000000
+          {% endif %}
```
Comment (Stefan-Ethernal) [inline .github/workflows/deploy.nightly.devnet.yml]: Don't need this either if it turns out that the `EIP1559Burn` is not needed. Agree?

File: .github/workflows/deploy.nightly.devnet.yml (hunk 1 above)
Comment (gatsbyz) [reply .github/workflows/deploy.nightly.devnet.yml]: oops this is deleted

**Review by Stefan-Ethernal — APPROVED** [review]

File: .github/workflows/deploy.nightly.devnet.yml [hunk 4]
```diff
@@ -80,6 +80,8 @@ jobs:
         env:
           TF_VAR_deployment_name: ${{ secrets.TF_VAR_DEPLOYMENT_NAME }}
           TF_VAR_owner: ${{ secrets.TF_VAR_OWNER }}
+      - name: Retrieve state file from s3
+        run: aws s3 cp s3://polygon-edge-devnet-tf-states/state/${{ secrets.TF_VAR_DEPLOYMENT_NAME }} state.json
```
Comment (vcastellm) [inline .github/workflows/deploy.nightly.devnet.yml:84]: What is the state file?

File: .github/workflows/manual.deploy.nightly.devnet.yml [hunk 5]
```diff
@@ -3,9 +3,9 @@
 
 name: Nightly DevNet Workflow (Manual)
 on:  # yamllint disable-line rule:truthy
-  # push:
-  #   branches:
-  #     - jesse/fix-nightly-build
+  push:
+    branches:
+      - jesse/fix-*
```
Comment (vcastellm) [inline .github/workflows/manual.deploy.nightly.devnet.yml:8]: Should this be removed?

**Review by vcastellm — APPROVED** [review]

**Review by vcastellm — APPROVED** [review]

File: .github/workflows/deploy.nightly.devnet.yml (hunk 4 above)
Comment (gatsbyz) [reply .github/workflows/deploy.nightly.devnet.yml:84]: this should be deleted

File: .github/workflows/manual.deploy.nightly.devnet.yml (hunk 5 above)
Comment (gatsbyz) [reply .github/workflows/manual.deploy.nightly.devnet.yml:8]: i can remove this anytime. i feel like i keep adding this back in whenever i fix the nightly so i just kept it in for now. i'll remove it soon.

**Pull request merged**